tools/rln_keystore_generator/rln_keystore_generator.nim

```diff
@@ -24,7 +29,48 @@ when isMainModule:
   
   debug "configuration", conf = $conf
 
-  # initialize keystore
+  # 2. initialize rlnInstance
+  let rlnInstanceRes = createRLNInstance(d=20, 
+                                         tree_path = genTempPath("rln_tree", "rln_keystore_generator"))
+  if rlnInstanceRes.isErr():
+    error "failure while creating RLN instance", error=rlnInstanceRes.error()
+    quit(1)
+  
+  let rlnInstance = rlnInstanceRes.get()
+
+  # 3. generate credentials
+  let credentialRes = rlnInstance.membershipKeyGen()
+  if credentialRes.isErr():
+    error "failure while generating credentials", error=credentialRes.error()
+    quit(1)
+
+  let credential = credentialRes.get()
+  debug "credentials", idTrapdoor = credential.idTrapdoor.inHex(), 
+                       idNullifier = credential.idNullifier.inHex(),
+                       idSecretHash = credential.idSecretHash.inHex(),
+                       idCommitment = credential.idCommitment.inHex()
+
+  # 4. write to keystore
+  ## TODO: after hooking up to the OnchainGroupManager, 
+  ## obtain chainId and treeIndex from the contract
+  let keystoreCred = MembershipCredentials(
+    identityCredential: credential,
+    membershipGroups: @[MembershipGroup(
+      membershipContract: MembershipContract(
+        chainId: "1155511",
```

Comment:
Shall we have a comment on this value? Can this come from config as well?

Reply:
This is a dummy value that will be replaced with the next iteration!